Libraries/Components/Navigation/NavigatorIOS.ios.js

```diff
@@ -25,6 +25,8 @@ var invariant = require('fbjs/lib/invariant');
 var logError = require('logError');
 var requireNativeComponent = require('requireNativeComponent');
 
+var keyMirror = require('fbjs/lib/keyMirror');
```

Comment:
I was thinking the same but then I thought - adding `var`s is just going to increase the work for people migrating later. We might as well introduce `const` gradually.